security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/LoadActionBuilder.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+/*
+ * @author Anne Tilloy <anne.tilloy@rte-france.com>
+ */
+public class LoadActionBuilder {
+
+    private String id;
+    private String loadId;
+    private Boolean relativeValue;
+    private Double activePowerValue;
+    private Double reactivePowerValue;
+
+    public LoadAction build() {
```

Comment:
maybe a check on loadId can be add ?

Reply:
I am going to do that in another PR for all actions.